challenges/objects-arrays.js

```diff
@@ -43,26 +67,32 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+const universities = graduates.map(function(element) {
```

Comment:
Your answer is a correct use of .map, however you were instructed to NOT use any array methods. A for loop would have been the correct answer.